R/shiny.R

```diff
@@ -266,24 +266,36 @@ dataTableProxy = function(
 #' @param selected an integer vector of row/column indices, or a matrix of two
 #'   columns (row and column indices, respectively) for cell indices; you may
 #'   use \code{NULL} to clear existing selections
+#' @param ignore.selectable when \code{TRUE} (the default), the "selectable"
+#'   range that's specified by \code{selection = list(selectable= )} will be
+#'   ignored; otherwise, you can't select the "non-selectable" range.
 #' @rdname proxy
 #' @export
-selectRows = function(proxy, selected) {
-  invokeRemote(proxy, 'selectRows', list(I_null(as.integer(selected))))
+selectRows = function(proxy, selected, ignore.selectable = TRUE) {
```

Comment:
I wonder if `TRUE` is a reasonable default. If certain rows were specified to be not selectable in `datatable()`, we should probably respect that specification by default (because there might be good reasons why they are not selectable). What do you think?

Reply:
> because there might be good reasons why they are not selectable

I agree.